geopandas/io/arrow.py

```diff
@@ -638,27 +688,33 @@ def _read_parquet(path, columns=None, storage_options=None, **kwargs):
     )
 
     path = _expand_user(path)
+    schema, metadata = _read_parquet_schema_and_metadata(path, filesystem)
+    geo_metadata = _validate_metadata(metadata)
+
+    bbox_filter = _get_parquet_bbox_filter(geo_metadata, bbox) if bbox else None
+
+    if_bbox_column_exists = _check_if_bbox_column_in_parquet(geo_metadata)
```

Comment:
seems like this should throw if `read_bbox_column` is true and `if_bbox_column_exists` is false